api/graphql/schema.graphql

```diff
@@ -351,6 +354,26 @@ input TopicSubscriptionInput {
   newPostCount: Int
 }
 
+input CommunityInput {
+  name: String
+  description: String
+  location: String
+  welcomeMessage: String
+  bannerUrl: String
+  avatarUrl: String
+  settings: CommunitySettingsInput
+  leaderId: ID
+  betaAccessCode: String
```

Comment:
we don't need to expose this functionality yet (and in any case it'll be called something else)